lib/client-assets.php

```diff
@@ -709,8 +709,8 @@ function gutenberg_extend_settings_custom_units( $settings ) {
  * Register default pattern categories if not registered in Core already.
  */
 if ( class_exists( 'WP_Block_Pattern_Categories_Registry' ) ) {
-	register_block_pattern_category( 'text', array( 'label' => __( 'Text', 'gutenberg' ) ) );
-	register_block_pattern_category( 'hero', array( 'label' => __( 'Hero', 'gutenberg' ) ) );
+	register_block_pattern_category( 'text', array( 'label' => _x( 'Text', 'Block pattern category', 'gutenberg' ) ) );
+	register_block_pattern_category( 'hero', array( 'label' => _x( 'Hero', 'Block pattern category', 'gutenberg' ) ) );
```

Comment:
Why only add the context to these two categories? I'd add them to all for consistency.

Reply:
I didn't know whether you meant all or just these two originally. The downside is the potential string duplication. I'll update